.eslintrc.js

```diff
@@ -0,0 +1,29 @@
+module.exports = {
+	env: {
+		commonjs: true,
+		es2021: true,
+		node: true
+	},
+	extends: 'eslint:recommended',
+	parserOptions: {
+		ecmaVersion: 12
+	},
+	rules: {
```

Comment:
Maybe add https://eslint.org/docs/rules/eqeqeq ?

Reply:
After fully reviewing, you seem to prefer template literals over string concatenations, so I would suggest adding:
https://eslint.org/docs/rules/prefer-template

Reply:
It would be very good to just use [eslint-config-standard](https://github.com/standard/eslint-config-standard), which already has all the best rules.

Reply:
It already uses `eslint:recommended`

Reply:
Yeah, I've seen it. But instead of adding a bunch of rules in addition to `eslint:recommended`, it would be much better to just use `eslint-config-standard`, which already has `eqeqeq`, `prefer-template` and much more the best rules. Isn't it?

Reply:
I don't think all style needs to be codified, and I don't think eslint is all that appropriate for things that aren't merely style. === for example, configuring eslint to complain about all uses of === is just as likely to mislead an inexperienced developer as it is to save them, if they don't know what the difference is. A lot of the time, you actually *want* loose equality checks.

Reply:
See: https://github.com/DoctorMcKay/node-steamcommunity/pull/230/commits/62f4651e805f52600d0daa66f14b7e4ef12c905b

Reply:
As for prefer-template, I think old-school concatenation has its place. I don't think I'd prefer enforcing template literals everywhere.